frontend/src/pages/LanSharePage/LanSharePage.tsx

```diff
@@ -56,30 +56,61 @@ export const LanSharePage: React.FC = () => {
   const history = useHistory()
   const css = useStyles()
   const [currentHost, setCurrentHost] = useState((connection && connection.host) || IP_PRIVATE)
-
-  if (!connection || !service) return null
+  const [error, setError] = useState<string>()
+  const [bindIP, setBindIP] = useState(IP_OPEN)
```

Comment:
```suggestion
```
I don't think you want a new state for `bindIP`. `bindIP` is the same as `currentHost` I think.